tests/e2e/functional_test.go

```diff
@@ -290,6 +290,8 @@ var _ = Describe("Functional e2e", Serial, func() {
 
 	})
 
+	time.Sleep(2 * time.Second)
```

Comment:
there's a 2 second sleep between any changes we do and I'd previously forgotten it in this spot - this is just to have a little more "real world timing" in the test - this was thought to make things a little easier on numaflow - we can potentially take it out later